lib/src/date-input/DateInput.tsx

```diff
@@ -36,28 +41,35 @@ const DxcDateInput = React.forwardRef<RefType, DateInputPropsType>(
       tabIndex,
     },
     ref
```

Comment:
The `ref` prop is being declared but never connected with any component at all. If the user tries to use it it will be `undefined`!